pkgs/development/ruby-modules/gem/default.nix

```diff
@@ -184,6 +184,7 @@ stdenv.mkDerivation (attrs // {
       --install-dir "$GEM_HOME" \
       --build-root '/' \
       --backtrace \
+      --no-env-shebang \
```

Comment:
Are there packages, which call "gem install" from there build system?
Or is this not a problem, because those shebangs gets fixed later?